src/game/common/system/quotedprintable.cpp

```diff
@@ -49,3 +49,47 @@ Utf8String Ascii_String_To_Quoted_Printable(Utf8String string)
     dest[count] = '\0';
     return dest;
 }
+
+int Hex_Digit_To_Int(char c)
+{
+    if (c <= '9' && c >= '0') {
+        return c - '0';
+    }
+
+    if (c <= 'f' && c >= 'a') {
+        return c - 'W';
```

Comment:
```c++
c - 'a' - 10
```
Compiler will make constant value 'W' out of that.

Reply:
fixed.